js/chat_app/src/services/chatServices.js

```diff
@@ -0,0 +1,38 @@
+export const chatServices = {
+  createChat: async (title, authToken) => {
+    try {
+      const data = await fetch('/api/chats', {
+        method: 'POST',
+        headers: {
```

Comment:
You duplicate it in every request. I would create some generic method or object and the will take all shared data from 1 place instead of duplicating it every time